.github/workflows/workflow.yml

```diff
@@ -99,6 +99,50 @@ jobs:
           retry_on: error
           command: npx nx affected --target=test:ci --exclude=@netlify/build --parallel=3
 
+  test-edge-bundler-integration:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        node-version: [14.16.0, '*']
+        exclude:
+          - os: macOS-latest
+            node-version: 14.16.0
+          - os: windows-latest
+            node-version: 14.16.0
+      fail-fast: false
+    steps:
+      # Sets an output parameter if this is a release PR
+      - name: Check for release
+        id: release-check
+        # For windows we have to use $env:
+        run: |-
+          echo "IS_RELEASE=true" >> $GITHUB_OUTPUT
+          echo "IS_RELEASE=true" >> $env:GITHUB_OUTPUT
+        if: ${{ startsWith(github.head_ref, 'release-') }}
```

Comment:
nit: we could extract this into an own custom action that checks for releases as we have this on every workflow job.

Reply:
True, but let's do that in a separate PR if it will ever bother us that much.